spec/views/posts_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :view do
+  before :each do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    @user.posts.create(title: 'Title', text: 'Body')
+    visit user_post_path(@user, @user.posts.first)
+  end
+
+  it 'Should display post\s title' do
+    expect(page).to have_content('Title')
+  end
+  it 'Should display post author name' do
+    expect(page).to have_content('Tom')
+  end
+
+  it 'should display comment counter' do
+    expect(page).to have_content('Comments')
+  end
+
+  it 'should display likes counter' do
+    expect(page).to have_content('Likes')
+  end
+
+  it 'Should display text' do
+    expect(page).to have_content('Body')
+  end
+
+  it 'Should display comment user name' do
+    @user.posts.first.comments.create(text: 'Comment sample', author_id: @user)
+    visit user_post_path(@user, @user.posts.first)
+    expect(page).to have_content('Tom')
+  end
+
+  it 'Should content See all posts button' do
+    visit user_path(@user.id)
+    expect(page).to have_content('Title')
+    expect(page).to have_content('See all posts')
+  end
+end
```

Comment:
- [ ] Please kindly include tests that assert the following 
  - You can see the first comments on a post.
  - You can see how many likes a post has.
  - You can see a section for pagination if there are more posts than fit on the view
  - Redirects to the post's show page when you click on a post.

Reply:
Thank you very much @AnselemOdims, let me do it right away!